guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,436 @@
+6. DICOM router sends the study further to PACS 
+7. PACS or the AI processing engine notifies RIS about the AI diagnosis being ready in the radiology worklist. 
+8. The radiologist and/or physician responsible for ordering the study or reviewing the results sees the case come up on top of their worklist with both the original DICOM image and the AI provided scoring.  
+
+**Clinical Use Case:** prospective diagnostics
+
+**Computational Workload Type:** [Asynchronous](#asynchronous-computational-workload)
+
+**Clinical workload:** 1,500 studies per day across all modalities with less 18 minutes per year of time required for manual intervention and less than 1% failures in processing.  Study size N x M x K depending on modality and requested study type.  See Table for estimates of study size to modality and study type mapping.
+
+**Importance to Clinical AI Deployment:** high, the implementation of AI as part of scheduled radiology diagnosis will have a significant impact on patient care
+
+**Figure:** Scheduled Radiology Sequence
+
+![MONAI_Scheduled Radiology](assets/MONAI_Scheduled_Radiology.jpg)
+
+#### Interactive Diagnostics During Image Reading  {#interactive-diagnostics-during-image-reading}
+
+While a radiologist is reading a particular study, they request an AI algorithm to analyze it.
+
+A time delay of 10 secs might be acceptable in _edge cases_, where the radiologist themselves needs a second opinion from fellow radiologists. 
+
+**Classifying nodules (localised anomaly) in a 3D scan**
+
+Imagine a 3D image where a radiologist only wants to get an inference on a small section of an image. The small section may be free hand drawn by the radiologist a selected 3D cube (or a 2D rectangular region). An inference will be done only in the selected region. Annotating this region will generate a DICOM SR object and only the SR object will be sent to the inference engine. The inference will be computed and a SR object will be sent back.   
+
+**Inference on only a single series in a 4D image**
+
+Consider a scenario where AI segmentation is done only on the Left Ventricular Myocardium (LVM) only on the systolic and diastolic phases of the cardiac cycle.  The problem here is both image segmentation and potentially using AI to select the ROI and the correct sub-images to process. 
+
+Assume for now the radiologist can select which series he/she wants to perform an inference. After selection the radiologist can just send the series number (or identifier) and inference result (dicom seg) object can be sent as a reply. 
+
+In the both the above mentioned cases, we are assuming that DICOM adapter in the AI platform pulls the images from the PACS with some identifier and the AI platform send just a lightweight DICOM SR object. We are also assuming a fast communication between PACS and the system where MONAI is deployed. Maybe the appropriate images are prefetched on the system (from PACS) where MONAI is deployed. 
```

Comment:
I'm not understanding this use case, and again think precise terminology will help. "4D" images are often stored in a single DICOM series, so I'm not sure "only a single series in a 4D image" makes much sense. Selecting certain instances/frames would certainly make sense.

Reply:
I did not know that the 4D images are stored as single dicom series. What I meant was that often you are not making the inference on the whole series but only one image at a certain timepoint. I had to write some code to separate the 4D images into 3D images using pydicom. This separation is done using the relevant dicom tags. Maybe something similar can be implemented on the back end which is a part of preprocessing the image. I don't necessarily know how to do this. But I dont think we should send a 4D image (from the viewing station )for inference when the inference needs to be done on only a single time point.

Reply:
Simplified for now and clarified the assumptions